W-15612083-Run-VALE-on-latest-Studio-repo-CP (#524)
mulesoft/docs-studio: modules/ROOT/pages/change-jdk-for-studio.adoc

You can find the JDK version used by Studio under *About Anypoint Studio* > *Installation Details*.

> If you must start Studio using a different JDK, follow the following instructions based on your operative system:

Reviewer: ```suggestion
To start Studio using a different JDK, follow the instructions based on your operating system:
```

Author: OK! Thanks!
